src/Hl7.Fhir.Base/Model/PocoNodeExtensions.cs

```diff
@@ -335,4 +346,20 @@ internal static string SerializeToString(this PocoNode pn, bool pretty)
 
         return serializer.SerializeToString(pn.Poco, pretty, rootName: rootName);
     }
+
+    /// <summary>
+    /// Converts the specified PocoNode instance to its JSON representation.
+    /// </summary>
+    /// <param name="pn">The PocoNode instance to serialize.</param>
+    /// <param name="pretty">Indicates whether the JSON output should be formatted for readability.</param>
+    /// <returns>A JSON string representing the given PocoNode.</returns>
+    public static string ToJson(this PocoNode pn, bool pretty)
+    {
+#pragma warning disable CS0618// Type or member is obsolete
```

Comment:
It looks strange to use obsolete calls in new code. Especially now you're making it public and will have to support this method indefinitely.
Will you ever be able to solve that?

Reply:
Will add those tests in a separate tasks.